rindexer_rust_playground/src/rindexer_lib/indexers/rindexer_playground/uniswap_v3_pool_filter.rs

```diff
@@ -1,151 +1,93 @@
 #![allow(non_snake_case)]
-use std::{path::PathBuf, sync::Arc};
```

Comment:
files got messed up maybe just need to add formatter? we should generate the file for people nicely

Reply:
Fixed this, had to move some things around as rustfmt couldn't parse the existing structure with an awaited nested callback